docs/ldml/tr35.md

```diff
@@ -2225,26 +2229,29 @@ For example, if und_AF ⇒ fa_Arab_AF, then:
 There are a few exceptions to this goal:
 * A 'denormalized' subtag changes to the normalized form, except for certain denormalized language subtags such as 'iw' (for 'he' = Hebrew) which may occur in both the 'from' and 'to' fields of the data.
 This allows for implementations that use those denormalized subtags to use the data with only minor changes to the operations.
-* A macroregion (such as West Africa = 011) may change to a specific country (Nigeria = NG). 
+* A macroregion (such as West Africa = 011) _may_ change to a specific country (Nigeria = NG). 
 
 **_Remove_** _**Likely Subtags:** Given a locale, remove any fields that Add Likely Subtags would add._
 
-The reverse operation removes fields that would be added by the first operation.
+The reverse operation removes fields that could be added by the first operation.
 
-1. First get max = AddLikelySubtags(inputLocale). If an error is signaled, return it.
-2. Remove the variants from max.
+1. First get max = AddLikelySubtags(inputLocale). If an error is signaled AddLikelySubtags, signal that same error and stop.
+2. Remove the variants and extensions from max.
 3. Get the components of the max (_languagemax_, _scriptmax_, _regionmax_).
 4. Then for _trial_ in {_languagemax_, _languagemax_regionmax_, _languagemax_scriptmax_}
-   * If AddLikelySubtags(_trial_) = max, then return _trial_ + variants.
-5. If you do not get a match, return max + variants.
+   * If AddLikelySubtags(_trial_) = max, then return _trial_ + variants + extensions.
```

Comment:
Since any call to AddLikelySubtags(_trial_) may get an error to indicate no match, we need to define what the algorith should do in that case. Without specifying that may imply the error should be propagated up and terminate the lookup loop. We should explicitly mention the error from the calling of AddLikelySubtags(_trial_) should not be propogate up but ignored and consider no matching only and continue the next look up.

Reply:
For example, consider the case of 
Remove Likely Subtags ("qaa-Cyrl-CH-u-nu-thai") here, what will happen?